src/Argument.h

```diff
@@ -56,13 +56,11 @@ struct Argument {
      * By default, they are left unset, implying "no default, no min, no max". */
     Expr def, min, max;
 
-  int stride_multiples[4];
```

Comment:
So, I realized that I could attempt to do this rebase myself. I tried to do it, and it is very difficult. There are many, many conflicts.

I think the best thing to do here is to just make new commits manually for the changes you want, on a new branch starting from the github.com/halide/Halide hexagon branch. I don't think there actually are that many. I'll go through the diff on this PR and make comments on the changes that should *not* be made.